src/Evolution/Systems/GeneralizedHarmonic/Tags.hpp

```diff
@@ -53,17 +53,66 @@ struct ConstraintGamma2 : db::SimpleTag {
   using type = Scalar<DataVector>;
   static std::string name() noexcept { return "ConstraintGamma2"; }
 };
+/*!
```

Comment:
Consistent formatting would have a line break between the previous struct and this doxygen.

Reply:
added